lib/galaxy/datatypes/converters/sam_to_bam.py

```diff
@@ -20,6 +21,17 @@ def cleanup_before_exit( tmp_dir ):
         shutil.rmtree( tmp_dir )
 
 
+def getSamtoolsVersion():
+    cmd = 'samtools --version'
```

Comment:
This will just raise an error on samtool 0.1.19, so if you return `None` further down, LooseVersion will raise an error.